src/clients/klist/klist.c

```diff
@@ -560,32 +562,28 @@ check_ccache(krb5_ccache cache)
 }
 
 /* Return true if princ is the local krbtgt principal for local_realm. */
-krb5_boolean
+static krb5_boolean
 is_local_tgt(krb5_principal princ, krb5_data *realm)
 {
     return princ->length == 2 && data_eq(princ->realm, *realm) &&
         data_eq_string(princ->data[0], KRB5_TGS_NAME) &&
         data_eq(princ->data[1], *realm);
 }
 
-char *
-etype_string(enctype)
-    krb5_enctype enctype;
+static char *
+etype_string(krb5_enctype enctype)
 {
     static char buf[100];
-    krb5_error_code retval;
+    krb5_error_code ret;
 
-    if ((retval = krb5_enctype_to_name(enctype, FALSE, buf, sizeof(buf)))) {
-        /* XXX if there's an error != EINVAL, I should probably report it */
```

Comment:
I'm not sure it's enough of a concern to justify the comment.  krb5_enctype_to_name() can't return an error other than EINVAL unless you pass it an overly short buffer (and 100 bytes is plenty for any enctype name), and there's no reason to expect that to change.